app/routes/pages/components/VisionPage.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router';
+import DisplayVision from './DisplayVision';
+import { Content } from 'app/components/Content';
+import styles from './VisionPage.css';
+
+class VisionPage extends Component {
+  render() {
+    return (
+      <Content>
+        <Link to={'/pages/about'}>-> Landing page</Link>
+
+        <DisplayVision />
+      </Content>
+    );
+  }
+}
```

Comment:
This could be written as a functional stateless component, unless you are planning adding state here later :)
```
const VisionPage = () => (
  <Content>
  ...
  </Content>
)
```